barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.hpp

```diff
@@ -673,7 +673,8 @@ class UltraCircuitBuilder_ : public CircuitBuilderBase<typename Arithmetization:
     UltraCircuitBuilder_(const size_t size_hint,
                          auto& witness_values,
                          const std::vector<uint32_t>& public_inputs,
-                         size_t varnum)
+                         size_t varnum,
+                         bool recursive)
```

Comment:
Minor nit but can we default this a la:
```suggestion
                         bool recursive = false)
```
This way we don't even need to make any change in GoblinUltraCircuitBuilder. Any time we can avoid a floating "true/false" in a constructor I like to do that